ruby-hacktoberfest-2023/Find_Factorial.rb

```diff
@@ -0,0 +1,11 @@
+n=gets.to_i
```

Comment:
User Interaction Update: You can print a message on the screen, asking the user to enter a number.